src/OneExplorer.ts

```diff
@@ -216,7 +216,8 @@ export class OneExplorer {
 
     let runCfgDisposal =
         vscode.commands.registerCommand('onevscode.run-cfg', (oneNode: OneNode) => {
-          handleRunOnecc(oneNode.node.uri, new Logger);
```

Comment:
(This is an extra discussion)
(Q) Actually, I wonder why this line is critical to change. Is there any reason to share the same logger? The logger firstly looked like a log format provider. But it isn't? 

1. If we need to use the same logger across our vscode extension, why don't we provide a global context?
2. If loggers are not to be shared, why don't we just make it disposable as our inner project does?

I find it is designed to uses createOutputChannel('ONE-VSCode'), but I cannot see the difference between new Logger to shared logger.

Reply:
That is a good point. I haven't really thought over logger and maybe we could consider after this release.
I will create an issue.

For this change,
I am not sure if having multiple `vscode.OutputChannel` is thread-safe. Maybe using one instance could be a bit safer.

Reply:
#562 was created for further discussion.